18F/identity-idp
# Remove unused and redundant alert icons

## 🛠 Summary of changes

Removes icon images which either duplicate design system icons ([`alerts/success.svg`](https://github.com/18F/identity-design-system/blob/main/src/img/alerts/success.svg)) or are unused.

## 📜 Testing Plan

Verify no regression in appearance of success icon in affected screens:

1. Go to http://localhost:3000
2. Sign in
3. Go to http://localhost:3000/verify
4. Continue process until step indicator reflects at least one completed step
5. Observe the success icon appears as expected

Verify there are no other references to removed images.

* `warning.svg` is used, but is [referenced](https://github.com/18F/identity-idp/blob/7c0084aed0fc8622c8ff065028a475d4dbdb8196/app/components/alert_icon_component.rb#L3) from [`images/status/warning.svg`](https://github.com/18F/identity-idp/blob/main/app/assets/images/status/warning.svg), not removed here
* `unphishable.svg` is used, but is [referenced](https://github.com/18F/identity-idp/blob/7c0084aed0fc8622c8ff065028a475d4dbdb8196/app/components/badge_component.html.erb#L2) from [design system `alerts/unphishable.svg`](https://github.com/18F/identity-design-system/blob/main/src/img/alerts/unphishable.svg) (observe "alerts" folder vs. "alert" removed file here)
